sites/public/page_content/terms.md

```diff
@@ -0,0 +1,86 @@
+You must understand that the Website, including all Content and Services, does NOT constitute and is NOT intended to serve as legal or financial advice in any capacity, or as advice for any particular purpose, and you agree to not use or rely upon the Website as such.
+You must acknowledge, understand and agree that although Website, including certain Content and Services, may describe various governmental functions or activities or summarize governmental information, including public laws, rules, regulations, orders, notices, records, or other publications, NONE of the Content constitutes or represents, and you agree to not use or rely upon the Website, including any Content and Services, an official representation of governmental activities or official governmental publication, unless it is clearly and expressly marked as such.
+You must understand and agree that although certain Services may enable or facilitate Users’ engagement or other form of communication with governmental entities, departments or officials, NONE of the Services constitutes or represents, and must not be used or relied upon as, an official means of communication for any governmental purpose. Acceptance of any engagement or communication made by means of the Website or any Services is at the City’s sole discretion.
+You may provide links to the Website, whether from another website, social media post, or other publication in any medium. You agree that doing so in any instance is subject to all of the following conditions: (a) the link directs to the Website’s homepage, located at “https://-----------.com”; (b) you do not remove or obscure, by framing or otherwise, any portion of the Website, including advertisements, links to these Terms, copyright notices, or other notices on the Web Site; (c) you provide prior written notice to the City of such link by sending an email to: detroithomeconnect@detroitmi.gov and (d) you acknowledge that the City may request that you refrain or discontinue providing links to the Website at any time and for any reason in its sole and absolute discretion, and you agree to refrain from or immediately discontinue providing links to the Website upon such request by the City.
```

Comment:
Looks like this link is missing, and I see it is also missing from the issue text, curious @kathyccheng if this is meant to be empty?

Reply:
@ColinBuyck and @emilyjablonski , I updated this to be `https://homeconnect.detroitmi.gov`